docs: add warning about Flux suspension after detachment (#4228)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.2/clusters/delete-cluster/index.md

When you attach a cluster to Kommander that was not created with Kommander, you can later disconnect it. This does not alter the running state of the cluster, but simply removes it from the Kommander UI. User workloads, platform services, and other Kubernetes resources are not cleaned up at detach.

<p class="message--warning"><strong>WARNING: </strong>

> After successfully detaching the cluster, manually disconnect the attached cluster's Flux installation from the management Git repository. Otherwise, changes to apps in the managed cluster's workspace will still be reflected on the cluster you just detached: <code>kubectl -n kommander-flux patch gitrepo management -p '{"spec":{"suspend":true}}' --type merge</code>.

Reviewer: The user will need to specify the correct kubeconfig and/or context for the detached cluster, right?

Reviewer 2: If yes, maybe we can reuse this: 

Ensure your `dkp` configuration references the cluster, where you want to run the upgrade. You can do this by setting the `KUBECONFIG` environment variable [to the appropriate kubeconfig file’s location](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/). An alternative to initializing the `KUBECONFIG` environment variable is to use the `–kubeconfig=cluster_name.conf` flag.

Author: Thanks @gvasquezvargas! I added these sentences to the info box.

> After successfully detaching the cluster, manually disconnect the attached cluster's Flux installation from the management Git repository. Otherwise, changes to apps in the managed cluster's workspace will still be reflected on the cluster you just detached: <code>kubectl -n kommander-flux patch gitrepo management -p '{"spec":{"suspend":true}}' --type merge</code>.

Reviewer 3: ```suggestion
After successfully detaching the cluster, manually disconnect the attached cluster's Flux installation from the management Git repository. Otherwise, changes to applications in the managed cluster's workspace is still reflected on the cluster you just detached: <code>kubectl -n kommander-flux patch gitrepo management -p '{"spec":{"suspend":true}}' --type merge</code>.
```

When you attach a cluster to Kommander that was not created with Kommander, you can later disconnect it. This does not alter the running state of the cluster, but simply removes it from the Kommander UI. User workloads, platform services, and other Kubernetes resources are not cleaned up at detach.

<p class="message--warning"><strong>WARNING: </strong>

> After successfully detaching the cluster you need to manually disconnect the attached cluster's Flux installation from the management Git repository. Otherwise, changes to apps in the managed cluster's workspace will still be reflected on the then-detached cluster: <code>kubectl -n kommander-flux patch gitrepo management -p '{"spec":{"suspend":true}}' --type merge</code>

Reviewer 2: ```suggestion
After successfully detaching the cluster, disconnect the attached cluster's Flux installation from the management Git repository. Otherwise, changes to apps in the managed cluster's workspace will still be reflected on the then-detached cluster: <code>kubectl -n kommander-flux patch gitrepo management -p '{"spec":{"suspend":true}}' --type merge</code>.
```
